MyApplication/app/build.gradle

```diff
@@ -0,0 +1,29 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 29
+    buildToolsVersion "29.0.0"
+    defaultConfig {
```

Comment:
This is not templeApp gradle file you created some thing application named `MyApplicattion`
Please revert all these

Reply:
@Chromicle 
Yes , I overlooked this fault .
working on it.